src/deployment/bicep-templates/storageAccounts.bicep

```diff
@@ -33,7 +33,7 @@ var storageAccountFuncQueuesParams = [
   'update-queue'
   'webhooks'
   'signalr-events'
-  'custom-metrics'
```

Comment:
Is there any other code we need to remove by removing this queue?

Reply:
No. This was lingering from a few months ago and I just noticed it.